### PR TITLE
Extract conditional include of <filesystem> into a separate Fs.h

### PR DESCRIPTION
**Pull request by mbasmanova**

### CODE REVIEW

File: velox/common/base/tests/Filesystem.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*
```
Comment (majetideepak) [inline velox/common/base/tests/Filesystem.h:1]: Should we name this `FilesystemMacro.h`? Or simply put inside `Macros.h`. `Filesystem.h` seems like it probably contains a class definition to me.

**Review by majetideepak — COMMENTED** [review]
Nice cleanup! I did not know there were so many duplicates.

File: velox/common/base/tests/Filesystem.h (hunk 1 above)
Comment (mbasmanova) [reply velox/common/base/tests/Filesystem.h:1]: I was thinking about that, but there is no macro :-)

Comment (mbasmanova) [reply velox/common/base/tests/Filesystem.h:1]: Maybe Fs.h?

Comment (majetideepak) [reply velox/common/base/tests/Filesystem.h:1]: Yeah, Fs.h seems like a better choice.

**Review by majetideepak — APPROVED** [review]

**Pull request closed without merging**